testgrid/config/config.yaml

```diff
@@ -1990,8 +1990,40 @@ dashboards:
   - name: periodic-kubeadm-gce-1.6
     test_group_name: periodic-kubernetes-e2e-kubeadm-gce-1-6
 
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide no/nogo
+# signal for releases.
+# The first test/suite in the list is the primary suite for release purposes
+- name: release-1.6-blocking-hard
+  dashboard_tab:
+  - name: gce-1-6
+    test_group_name: ci-kubernetes-e2e-gce-release-1-6
+  - name: gce-serial-1-6
+    test_group_name: ci-kubernetes-e2e-gce-serial-release-1-6
+  - name: gce-slow-1-6
+    test_group_name: ci-kubernetes-e2e-gce-slow-release-1-6
+  - name: gce-reboot-1-6
+    test_group_name: ci-kubernetes-e2e-gce-reboot-release-1-6
+  - name: gce-scalability-1-6
+    test_group_name: ci-kubernetes-e2e-gce-scalability-release-1-6
+  - name: test-go-1-6
+    test_group_name: ci-kubernetes-test-go-release-1-6
+  - name: verify-1-6
+    test_group_name: ci-kubernetes-verify-release-1-6
+  - name: kubelet-1-6
+    test_group_name: ci-kubernetes-node-kubelet-1-6
+  - name: gke-serial-1-6
+    test_group_name: ci-kubernetes-e2e-gke-serial-release-1-6
+  - name: gke-1-6
+    test_group_name: ci-kubernetes-e2e-gke-release-1-6
+  - name: gke-slow-1-6
+    test_group_name: ci-kubernetes-e2e-gke-slow-release-1-6
+
+# These are the release "blocking" tests.
+# These need to be interpreted by a human
 # Matches the list on https://github.com/kubernetes/test-infra/issues/2029
-- name: release-1.6-blocking
+- name: release-1.6-blocking-soft
```

Comment:
Maybe I should just run with that and redefine blocking to be exactly what it should be.  The other tests that folks want to eyeball for whatever reason can be eyeballed elsewhere.  @kubernetes/kubernetes-release-managers can I get an ack on that so that this doesn't get reverted as soon as it goes through?  Let's decide now.